released/packages/coq-coquelicot/coq-coquelicot.3.0.3+8.11/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "guillaume.melquiond@inria.fr"
+homepage: "http://coquelicot.saclay.inria.fr/"
+dev-repo: "git+https://gitlab.inria.fr/coquelicot/coquelicot.git"
+bug-reports: "https://gitlab.inria.fr/coquelicot/coquelicot/issues"
+license: "LGPL 3-only"
```

Comment:
```suggestion
license: "LGPL-3.0-or-later"
```
From the headers in the .v files in the repo, the license looks to be "or later".

Reply:
Indeed, I was just looking at the copying file.